### PR TITLE
Fix DeltaF indexing edge case

### PR DESCRIPTION
**Pull request by JABeauchamp**
Fix edge case in deltaF calculations, where test unit average method was requested and two MUs had no overlapping region, causing an indexing issue.

### CODE REVIEW

File: openhdemg/library/pic.py [hunk 1]
```diff
@@ -179,7 +179,16 @@ def compute_deltaf(
...
+
+            # Collect which MUs were control vs test
+            if mu1_rcrt < mu2_rcrt or (mu1_rcrt == mu2_rcrt and mu1_drcrt > mu2_drcrt):
+                controlU = 1 
+            else:
+                controlU = 2
+
+            controlmu.append(mucombo[-1][controlU-1])
+            testmu.append(mucombo[-1][1-controlU//2])
+            continue  #
```
Comment (GiacomoValliPhD) [inline openhdemg/library/pic.py:192]: The problem seems addressed. This needs to be covered in the tests.

**Review by GiacomoValliPhD — APPROVED** [review]
The problem seems addressed. This needs to be covered in the tests.
